### ISSUE TITLE
Can't Reproduce the Same Accuracy (GradCAM_F1/model.py)

### ISSUE THREAD

**Issue by bharat3012** [rezacsedu/XAI_Cancer_Pred#1]
Dear Sir,

Thanks for this wonderful repository! I have been trying to reproduce the results for the GradCAM method. Like as you mentioned in the repo, we should run- python3 load_data.py (for preprocess), python3 model.py (train and valid accuracy).  But, firstly and unfortunately I got quite low accuracies when the model is trained for 50 epochs:
Epoch 46/50
20/20 [==============================] - 27s 1s/step - loss: 2.2263 - acc: 0.2986 - val_loss: 2.2967 - val_acc: 0.3482

Please let us know the reasons, I would like to work on this method.

Secondly, I am unable to find file: "data/pancan_genes.csv" used in load_data.py.

**Comment by rezacsedu** [comment]
Hi,

Thanks for your interest in our work. For the accuracy, I'd suggest you to iterate the training for a higher number of the epoch but if it doesn't help, it may be the case I uploaded the wrong data. I'll check but can't promise by when I can get back to you as I'm pretty overloaded ATM. 

For the "data/pancan_genes.csv" use the 'data/genes_List.csv' file instead, both files contain the same list of genes. 

By the way, I'm working on the journal version by extending this work. I'll make the updated repo open once I'm done with the submission. 


Best,
Karim

**Comment by bharat3012** [comment]
Hi Sir,

Thank you for your reply and help! Kindly let me know if dataset used is
named as fullDF.csv or something else? This small information would work
for me. Thanking you in advance.

Sincerely yours
Bharat Giddwani

**Comment by Y0901** [comment]
> Dear Sir,
> 
> Thanks for this wonderful repository! I have been trying to reproduce the results for the GradCAM method. Like as you mentioned in the repo, we should run- python3 load_data.py (for preprocess), python3 model.py (train and valid accuracy). But, firstly and unfortunately I got quite low accuracies when the model is trained for 50 epochs: Epoch 46/50 20/20 [==============================] - 27s 1s/step - loss: 2.2263 - acc: 0.2986 - val_loss: 2.2967 - val_acc: 0.3482
> 
> Please let us know the reasons, I would like to work on this method.
> 
> Secondly, I am unable to find file: "data/pancan_genes.csv" used in load_data.py.

Hi Sir

The author's data download link is currently invalid, could you please send the relevant data set you downloaded before?
 If possible, we can contact by email.

Thanks
